src/encoded/static/components/objectutils.js

```diff
@@ -884,10 +884,7 @@ export function computeAssemblyAnnotationValue(assembly, annotation) {
  */
 export const isFileVisualizable = file => (
     (file.file_format === 'bigWig' || file.file_format === 'bigBed')
-        && (file.file_format_type !== 'bedMethyl')
         && (file.file_format_type !== 'bedLogR')
```

Comment:
@emro file types like bedLogR were not enabled as they were deemed either obsolete or not use enough to be worth enabling. But in case any one of them need to be enabled, you can enable the genomic-reader (npm package that reads the bigbed file) by following this instruction I wrote- https://github.com/ENCODE-DCC/bigwig-reader#creating-custom-bigbed-formats